packages/engine/src/scene/behaviors/createCollidersFromScaneData.ts

```diff
@@ -0,0 +1,71 @@
+import { Behavior } from '../../common/interfaces/Behavior';
+import { Entity } from '../../ecs/classes/Entity';
+import { getComponent, createEntity, addComponent, removeComponent } from '../../ecs/functions/EntityFunctions';
+import { AssetLoader } from '../../assets/components/AssetLoader';
+import { TransformComponent } from "@xr3ngine/engine/src/transform/components/TransformComponent";
+import { ColliderComponent } from '../../physics/components/ColliderComponent';
+import { RigidBody } from '../../physics/components/RigidBody';
+import { addColliderWithoutEntity } from '../../physics/behaviors/addColliderWithoutEntity';
+import { Network } from '../../networking/classes/Network';
+import { createNetworkRigidBody } from '@xr3ngine/engine/src/templates/interactive/prefabs/NetworkRigidBody';
+import { addCollidersToNetworkVehicle } from '@xr3ngine/engine/src/templates/vehicle/prefabs/NetworkVehicle';
+
+export const createCollidersFromScaneData: Behavior = ( entity: Entity, args: any ) => { //{ data:string, type: string,}
```

Comment:
mispelling in createCollidersFromScaneData. should be createCollidersFromSc**e**neData as well as in file name

Reply:
sorry